app/services/report/export.rb

```diff
@@ -189,7 +189,7 @@ def next_quarter_forecasts
       def variance_data
         [
           variance_for_report_financial_quarter,
-          activity_presenter.comment_for_report(report_id: report_presenter.id)&.comment,
+          activity_presenter.comments_for_report(report_id: report_presenter.id).map(&:comment).join("|"),
```

Comment:
did you decide against using a `\n` in between comments?

Reply:
Oops - well spotted, I forgot that was a part of this work